lib/Twig/Extension.php

```diff
@@ -76,4 +76,12 @@ public function getGlobals()
     {
         return array();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
```

Comment:
It is marked as deprecated in the interface. Adding a `@deprecated` annotation here would generate deprecation notices when running the tests, that's why I've not added it here.